File: Algoritmica/Actividades/Actividad-III/Codigos/Tema_3/Tema3_03.cpp
//C++ code

void drawSpiral(Turtle myTurtle, int lineLen)
{
    // Compare with ActiveCode 2
    if (lineLen > 0)
    {
        myTurtle.forward(lineLen);
        myTurtle.right(90);
        drawSpiral(myTurtle, lineLen - 5);
    }
}

File: Algoritmica/Actividades/Actividad-III/Codigos/Tema_3/Tema3_03.py
#Tema3_03
import turtle

miTortuga = turtle.Turtle()
miVentana = turtle.Screen()

def dibujarEspiral(miTortuga, long_linea):
    if long_linea > 0:
        miTortuga.forward(long_linea)
        miTortuga.right(90)
        dibujarEspiral(miTortuga,long_linea-5)

dibujarEspiral(miTortuga,200)
miVentana.exitonclick()
